frameworks/PHP/webman/benchmark_config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "framework": "webman",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "db_url": "/db",
+      "query_url": "/queries/",
+      "update_url": "/updates/",
+      "fortune_url": "/fortunes",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "Postgres",
+      "framework": "None",
+      "language": "PHP",
+      "flavor": "PHP7",
+      "orm": "Raw",
+      "platform": "workerman",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "webman",
+      "notes": "",
+      "versus": "php"
```

Comment:
It's better compare with workerman.
So you will know the framework overhead.

Reply:
Do you mean to change `"display_name": "webman"` to `"display_name": "webman-workerman"` ?

Reply:
Do you mean to change `"display_name": "webman"` to `"display_name": "webman-workerman"` ?

Reply:
"versus": "workerman"

Reply:
OK, thank you for your advice.